py-doc-to-xml/docx_to_xml/doc1.py

```diff
@@ -136,26 +138,37 @@ def parse_semantic_domains(
         if value == "" or value[0] == "#":
             continue
 
-        # if the DocModel element starts with a semantic domain number, then
-        if is_semantic_domain_number(value):
+        # if the DocModel element starts with a semantic domain abbreviation, then
+        if is_semantic_domain_abbrev(value):
             # if the current semantic domain element is valid, add it to the list
             if current_semantic_domain.is_valid():
                 add_domain(semantic_domains, current_semantic_domain, use_warnings=use_warnings)
-            (domain_number, domain_title) = split_semantic_domain_line(value)
+            (domain_abbrev, domain_name) = split_semantic_domain_line(value)
             current_semantic_domain = SemanticDomain(
-                number=domain_number, title=domain_title, description="", questions=[]
+                abbrev=domain_abbrev, name=domain_name, description="", questions=[]
             )
         # else
         #   if the DocModel element starts with a question number, then
         elif is_question(value):
             # add the text to the list of questions
             (question_num, question_text) = split_question(value)
-            current_semantic_domain.questions.append(f"{question_num} {question_text}")
+            current_semantic_domain.questions.append(
+                DomainQuestion(question=f"{question_num} {question_text}"), words="", sentences=""
```

Comment:
the `)` needs to be moved from before `, words="", sentences=""` to after.

Reply:
Fixed (and verified).